src/site/_includes/components/Codepen.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {html} = require('common-tags');
```

Comment:
I'd suggest removing common-tags and returning a template string instead. I've found that common-tags does weird things with whitespace that sometimes makes the markdown parser grumpy. I didn't use it on d.c.c. and am thinking we should eventually try to remove it from web.dev.

Reply:
When I remove it the output breaks on the Iframe - not sure why exactly, so i keep it for now.

Reply:
I messed around with it a bit and was able to remove it and pushed a new commit that also added some docs.